crates/control/src/game_controller_filter.rs

```diff
@@ -71,9 +89,47 @@ impl GameControllerFilter {
                 hulks_team_is_home_after_coin_toss: game_controller_state_message
                     .hulks_team_is_home_after_coin_toss,
             });
+
+            self.last_contact.insert(*source_address, time);
+            let on_cooldown = self
+                .last_collision_warning
+                .is_some_and(|last_collision_warning| {
+                    time.duration_since(last_collision_warning)
+                        .expect("time ran backwards")
+                        .as_secs_f32()
+                        < 10.0
```

Comment:
Add a Paramalam Ding Dong? https://youtu.be/0HhA0Cghr4k

Same for the second constant below.

Reply:
Döne

Reply:
Nice choice of paramalam name 😂